test/myFTClient.spec.js

```diff
@@ -864,4 +872,159 @@ describe('myFT Client proxy', () => {
         .catch(done);
     });
   });
+
+    //new orgainic v3/kat methods
+    describe('Followed_by_Kat concepts', () => {
+
+      afterEach(done => {
+          nock.cleanAll();
+          done();
+      });
+
+
+      const relProps = Object.assign({}, myFT.followedProperties, {byTool: 'myFTClient.spec', isTest: true});
+      const userConcepts = require('./mocks/fixtures/userFollowedConcept');
+      const katConcepts = require('./mocks/fixtures/userFollowsConceptByKat');
+      const followedByKatRes = require('./mocks/fixtures/followByKatResp');
+    //  const groupConcepts = require('./mocks/fixtures/groupFollowedConcept');
+
+      it('Should set concept(s) follows by a user ', done => {
+          nock(baseUrl)
+            .post(`/kat/user/follows`)
+            .query({noEvent: 'true', waitForPurge: 'false'})
+            .reply(200, () => followedByKatRes);
+
+        myFT.addConceptsFollowedByKatUser(katConcepts.ids, katConcepts.subjects, relProps)
+          .then(addResp => {
+            expect(addResp).to.be.an('array');
+            expect(addResp[0][0][0]).to.have.deep.property('katRel.type', 'followed_by_kat');
```

Comment:
Out of interest, why are we looking at `[0][0][0]`?
Interested in why the API is returning nested arrays like this.

Reply:
Seems so. That's what I get in a postman reqest to the api. I'll take a look at myft-api and see if we can get a flatter response.